__integrations__/models/message.test.ts

```diff
@@ -242,4 +262,290 @@ describe("Models |> Message", () => {
 
     await context.dispose();
   });
+
+  it("should retrieve only one page without any continuation token when the default pageSize is greater than the number of results", async () => {
+    const context = await createContext(MESSAGE_MODEL_PK_FIELD);
+    await context.init();
+    const model = new MessageModel(context.container, MESSAGE_CONTAINER_NAME);
+
+    await createMessage(model, 1);
+    await createMessage(model, 2);
+
+    // get a page of messages by fiscal code
+    let results: DecodedFeedResponse<RetrievedMessage> = await model
```

Comment:
No, type is not inferred. This is the same reason because we discussed about a cast before. I wanted to enforce the type to make inferring work, but that did not work.

Reply:
Bad news, I think this to be an important feature for users of this API. If they'll need to manually specify the type, chances of error will be significant.

I'll try to figure out something

Reply:
It's very strange because that type seems pretty simple. Maybe TS doesn't like that disjointed union?

Reply:
@balanza the problem is not in the type itself but in the map here: `.then(ir => ir.value)`. ir.value is any.

Reply:
Tried a fix in https://github.com/pagopa/io-functions-commons/pull/198/commits/b981a60d6bf1c5dc73204489be4b7231d52083d2 and https://github.com/pagopa/io-functions-commons/pull/198/commits/9fb481ad4f58f72b808c48d5972948825ef1f53a